src/opencast.js

```diff
@@ -49,6 +49,10 @@ export class Opencast {
 
   updateGlobalOc = null;
 
+  IDS = new Map();
```

Comment:
As this isn't a constant, it should be `ids`. And please briefly document this attribute/field, in particular also mention an example value it will hold.